src/collective/cover/static/js/compose.js

```diff
@@ -96,7 +96,9 @@ $(document).ready(function() {
 
   TitleMarkupSetup();
 
-  if ($.fn.prepOverlay !== undefined) {
```

Comment:
changes on JS usually require an upgrade step to cook the resources; do you mind to check if we already have one in place?